src/app/App.js

```diff
@@ -56,7 +56,7 @@ function App() {
             product={product}
             handleClick={handleWishlistToggle}
             //coudln't deconstruct here for whatever reason
```

Comment:
Comment: Might have been the brackets... you need to wrap a destructure in brackets (even if theres only one argument)

Reply:
I've also added in strict type equality with the `===`, `==` would match both _id = '1' and _id = 1